packages/teams-js/src/public/meeting.ts

```diff
@@ -518,4 +518,25 @@ export namespace meeting {
     ensureInitialized(FrameContexts.sidePanel, FrameContexts.meetingStage);
     registerHandler('meeting.meetingReactionReceived', handler);
   }
+  /**
+   * nested namespace for Meeting Service
+   *
+   * @beta
+   */
+  export namespace meetingAppShareButton {
+    /**
+     * Allows an app to hide/show app share button in a meeting
+     * @param isGlobalAppShareButtonVisible is a boolean flag to set show or hide app share button
+     *
+     * @beta
+     */
+    export function setAppShareButtonVisibility(isGlobalAppShareButtonVisible: boolean): Promise<void> {
```

Comment:
If you are going with "appShareButton" as the subcapability name, this function might be better if it was named "setVisibility". Then a full call in code would read:

`meeting.appShareButton.setVisibility(visibility);`